pyne/ensdf_processing.py

```diff
@@ -104,3 +104,105 @@ def gabs(inputdict_unchecked):
     proc.communicate()[0]
     proc.stdin.close()
 
+def logft(inputdict_unchecked):
+    #NOTE: changed input file line length to 90 to support longer file paths in fortran source.
+    """
+    This program calculates log ft values for beta and electron-capture decay, average beta energies, 
+    and capture fractions.  (LOGFT readme)
+
+    Input Dictionary Required Key Pair Value:
+        input_data_set : path to input data file.
+        output_report : desired path to output report file.
+        data_table : path to data table.
+        output_data_set : desired path to output data set.
+
+    Output Dictionary Values:
+        Everything in input dictionary is returned if LOGFT completes successfully.
+    """
+    inputdict = {}
```

Comment:
As a general point, though not necessarily a blocker, in Python the convention of including the type name in the variable name is not advised.  Not a huge deal, but something to be aware of.